index.js

```diff
@@ -129,7 +134,11 @@ module.exports = string => {
 		throw new TypeError('Expected a string');
 	}
 
-	const indents = makeIndentsMap(string);
+	let indents = makeIndentsMap(string, true);
+	if (indents.size === 0) {
+		indents = makeIndentsMap(string, false);
+	}
```

Comment:
I'm protecting against introduction of another edge case here where a file truly is single space indented intentionally.  Since `makeIndentsMap` calculates _relative_ indents accounting for this inline would've sacrificed code clarity.  Trading off for a second parsing run here in the interest of limiting complexity.

Reply:
Can you add this as a code comment?